README.md

```diff
@@ -166,6 +171,9 @@ We also use the issue tracker to track features. If you have an idea for a featu
 [kops]: https://github.com/kubernetes/kops
 [kubicorn]: http://kubicorn.io/
 [slack]: https://kubernetes.slack.com/messages/cluster-api-openstack
+[zoom]: https://zoom.us/j/91405526408?pwd=bVBxempwVDBzejJGQlhSOGlXZnRxUT09
```

Comment:
Should this be visible to the public? Or would it be sufficient to link to the office hours notes where the zoom link can be found?

Reply:
In core CAPI they only have a link to the meeting, without the PW (which is available from the meeting notes doc).

It's probably not a good idea to have this public.

Reply:
It's already public: it's published in the public SIG-cluster-lifecycle calendar. It's presumably ok to publish it.